docs/source/common/production_inference.rst

```diff
@@ -1,41 +1,59 @@
 .. _production_inference:
 
+#######################
 Inference in Production
-=======================
-PyTorch Lightning eases the process of deploying models into production.
+#######################
 
+Since :class:`~pytorch_lightning.core.lightning.LightningModule` is just :class:`torch.nn.Module`, all the possible inferencing techniques for deploying PyTorch models to production
+apply here too. Lightning provides some helper methods to help you out with it.
 
-Exporting to ONNX
------------------
-PyTorch Lightning provides a handy function to quickly export your model to ONNX format, which allows the model to be independent of PyTorch and run on an ONNX Runtime.
+---------------
 
-To export your model to ONNX format call the ``to_onnx`` function on your Lightning Module with the filepath and input_sample.
+**********
```

Comment:
I would add a section on best practices with the introduction of the System. Meaning you lift nn.Module, so you don't need PyTorch Lightning dependency in production.

Reply:
I have introduced two separate sections, `With Lightning` and `Without Lightning`.  need your review :)